optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -919,7 +921,7 @@ def __init__(
         categorical_feature: str = "auto",
         early_stopping_rounds: Optional[int] = None,
         fpreproc: Optional[Callable[..., Any]] = None,
-        verbose_eval: Optional[Union[bool, int]] = True,
+        verbose_eval: Union[bool, int] = True,
```

Comment:
According to the [reference](https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.cv.html#lightgbm.cv), `verbose_eval` of `lightgbm.cv` accepts `None`. So could you revert this change?
(I noticed that the default value is not matched with `lightgbm.cv` although it is not in the scope of this PR.)
```suggestion
        verbose_eval: Optional[Union[bool, int]] = True,
```

Reply:
Done!